baaas-decision-fleet-manager/src/main/java/org/kie/baaas/dfm/app/manager/kafkaservice/KafkaServiceType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.kie.baaas.dfm.app.manager.kafkaservice;
+
+public enum KafkaServiceType {
+    OPERATE_FIRST("operate-first-kafka"),
+    MANAGED_KAFKA("managed-kafka"),
+    UNDEFINED("Undefined");
```

Comment:
I wonder if `DISABLED` is better than `UNDEFINED`. ?

Reply:
does sound better.